lib/qualtrics_api/directory.rb

```diff
@@ -0,0 +1,57 @@
+module QualtricsAPI
+  class Directory < BaseModel
+    values do
+      attribute :directory_id, String
+      attribute :name, String
+      attribute :contact_count, Integer
+      attribute :is_default, Boolean
+    end
+
+    def mailing_lists
+      mailing_list_collection.all
+    end
+
+    def create_mailing_list(directory_mailing_list)
+      mailing_list_collection.create_mailing_list(directory_mailing_list)
+    end
+
+    def transaction_batches
+      transaction_batch_collection.all
+    end
+
+    def create_transaction_batch
```

Comment:
We opted to skip passing in a locally created object with attributes because we currently don't have a need for setting transaction ids or creation date.